content/rule_book/roles_and_responsibilities.md

```diff
@@ -2,68 +2,96 @@
-    - Once all of the information in both the question and answer has been given, but something needs to be fixed, the quizmaster will say "Again".
-    - On a Situation type question, when the quizzer has completed the Quotation the quizmaster will say, "Quote is complete."
-    - On an INT or MA Question, if all of the information in the answer text has been given, but the quizzer is missing information in the question text, the quizmaster will say "More".
-    - On a Situation question, because the quotation is part of the question, the quizmaster will say "More" when required information from the quotation is missing.
-    - On a Q/Q2V, CVR/MACVR, CR/MACR question, if either the chapter or verse number were not completely finished when the quizzer jumped, the quizmaster will say "More".
-    - On a Reference type question, if the quizzer gives a reference question that is different than the written question, the quizmaster and answer judge must ensure that the reference question given is the exact same type as the written question.
-    - Consult with the answer judge(s) at any time they feel is necessary, provided it does not infringe upon the 30 seconds allotted the quizzer.
-    - If an explanation of a ruling is requested by a captain, the quizmaster will ask if any team wants to challenge before the explanation is given.
+- Rule immediately upon an obviously correct or incorrect answer if there is no indication from the answer judge to the contrary
+
+When a quizzer's response contains a required completion of the question and that completion be different than the written question, the quizmaster will discuss it with the answer judge. If the quizzer gives a reference question that is different than the written question, the quizmaster and answer judge must ensure that the reference question given is the exact same type as the written question.
+
+The quizmaster should consult with the answer judge(s) at any time they feel is necessary, provided it does not infringe upon the quizzer's question answer duration period.
+
+#### Quizmaster Prompts
+
+During the question answer duration period, the quizmaster will use only the following prompts.
+
+##### Pronoun Clarification
+
+If a quizzer has provided all the information in the answer, except the identification of a pronoun present which is part of the answer text, the quizmaster will ask the quizzer to identify the specific pronoun. For example: "Can you identify 'He'?" or "Can you identify 'They'?"
+
+The quizmaster is not allowed to say anything other than what the quizzer has already said when asking for the identification. A pronoun may need to be identified only if the antecedent is in context.
+
+Once the quizzer has completed all information in the answer, the quizmaster will prompt the quizzer to identify the specific pronoun, even if the quizzer has not yet completed all information in the question.
```

Comment:
Massive subtlety that I think could be a point of interest for those reviewing the rewrite:
I really don't know who does this. It's hard to know because quizzers almost never respond with the answer text first, and the remaining question text next.

Reply:
I feel like this should a v2 investigation. I don't like the idea of a QM being able to ask for a pronoun clarification prior to all the information in the question being provided because once the QM asks for a pronoun clarification and it's given, the QM then has to say something like, "Um, OK, so... Pronoun correct, but keep answering more now..."

Reply:
Should we nuke the "even if..." clause for v1?

Reply:
idk. I'm not sure what kind of bug this is. We kinda introduced it by being clear. I think most QMs now prompt for pronoun clarification when ALL Question+Answer information has been given.

Reply:
I agree. 535a257